lib/db/schema/patch-034-035.sql

```diff
@@ -0,0 +1,87 @@
+CREATE TABLE IF NOT EXISTS unblockCodes (
+  uid BINARY(16) NOT NULL,
+  unblockCode VARCHAR(32) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL,
+  UNIQUE INDEX unblockCodes_uid_code(uid, unblockCode)
```

Comment:
I didn't realize I could mark multiple columns as `PRIMARY KEY`, I assumed this was the best (and same thing).